src/tmdb.py

```
@@ -1639,29 +1639,37 @@ async def get_tmdb_localized_data(meta, data_type, language, append_to_response)
-                        localized_data = json.loads(content)
-                else:
+    if os.path.exists(filename):
+        try:
+            async with aiofiles.open(filename, 'r', encoding='utf-8') as f:
+                content = await f.read()
+                try:
+                    localized_data = json.loads(content)
+                except json.JSONDecodeError as e:
+                    console.print(f"[red]Warning: JSON decode error in {filename}: {e}. Creating new file.[/red]")
                     localized_data = {}
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)